### ISSUE TITLE
Customizable keyboard padding

### ISSUE THREAD

**Issue by nahuhh** [AnySoftKeyboard/AnySoftKeyboard#3251]
![Image_46.jpg](https://user-images.githubusercontent.com/50635951/154263953-478b88bd-34b7-46c5-bab8-34c85405fa95.jpg)

As can be seen on the image above, there is an unnecessary amount of padding below the keyboard when using system gestures. 

This request is basically to reopen #460

Padding options for left/right/bottom
Under UI > tweaks > tweaks 

- Left and right padding should stretch the keyboard
- bottom padding should shift the keyboard up/down (as to avoid changing the "height factor")

**Comment by Mabadman** [comment]
I agree with this. Unfortunately the layout for my language's keyboard is very cumbersome. It's ergonomically difficult to use some keys and I'd rather have some padding/margins to make it easier
